src/pygama/evt/modules/legend.py

```diff
@@ -0,0 +1,33 @@
+"""
+Module provides LEGEND internal functions
+"""
+from importlib import import_module
+
+
+def metadata(params: dict) -> list:
+    # only import legend meta data when needed.
+    # LEGEND collaborators can use the meta keyword
+    # While for users w/o access to the LEGEND meta data this is still working
+    lm = import_module("legendmeta")
+    lmeta = lm.LegendMetadata(path=params["meta_path"])
```

Comment:
Please allow for environment variable expansion in the config, you can just pass the expression to:

https://github.com/legend-exp/legend-pydataobj/blob/main/src/lgdo/lh5/utils.py

We should not hardcode paths here: https://github.com/legend-exp/legend-dataflow-config/pull/33

Reply:
Done in  b83c362